libraries/lib-time-and-pitch/StaffPadTimeAndPitch.h

```diff
@@ -15,7 +15,8 @@ class TIME_AND_PITCH_API StaffPadTimeAndPitch final :
 
 private:
    void BootStretcher();
-   const std::unique_ptr<staffpad::TimeAndPitch> mTimeAndPitch;
```

Comment:
Why remove `const`?  That's not necessary and having it here is useful information to a maintainer.